src/simulation/LoadGenerator.cpp

```diff
@@ -705,18 +706,27 @@ LoadGenerator::generateLoad(GeneratedLoadConfig cfg)
                         resources.instructions = rand_uniform<uint32_t>(
                             1, static_cast<uint32>(maxPerTx.getVal(
                                    Resource::Type::INSTRUCTIONS)));
-
-                        // Respect both contract size limit and TX write byte
-                        // limits including key overhead
-                        uint32_t const keyOverhead = 40;
-                        auto maxWasmSize =
-                            std::min(mApp.getLedgerManager()
-                                         .getSorobanNetworkConfig()
-                                         .maxContractSizeBytes(),
-                                     static_cast<uint32>(maxPerTx.getVal(
-                                         Resource::Type::WRITE_BYTES)) -
-                                         keyOverhead);
-                        wasmSize = rand_uniform<uint32_t>(1, maxWasmSize);
+                        auto const& appCfg = mApp.getConfig();
```

Comment:
Note that this change is obsolete since now we have https://github.com/stellar/stellar-core/pull/4248, I'll update this PR.